ext/drmgr/drmgr.h

```diff
@@ -721,6 +721,92 @@ DR_EXPORT
 ptr_uint_t
 drmgr_reserve_note_range(size_t size);
 
+/***************************************************************************
+ * EMULATION
+ */
+
+/**
+ * Holds data about an emulated instruction, typically populated by an emulation
+ * client and read by an observational client.
+ */
+typedef struct _emulated_instr_t {
+    size_t size;    /**< Size of this struct, used for API compatibility checks */
+    app_pc pc;      /**< The PC address of the emulated instruction. */
+    instr_t *instr; /**< The emulated instruction. */
```

Comment:
It is important to explain that ownership of \p instr is transferred to the label: \p instr will be freed when the label is freed.

Reply:
True, added a \note with an explanation.